src/SubstanceTokenizer.x

```diff
@@ -0,0 +1,87 @@
+{
+module SubstanceTokenizer where
+}
+%wrapper "basic"
+$digit = 0-9
+$alpha = [a-zA-Z]
+$symbol = [\=\+\-\*\/\:\∈\←\→\↑\↓\↔\↕\↖\↗\↘\↙\↚\↛\↮\⟵\⟶\⟷\<\>\|\;\~\`\!\#\%\&\*\±\§\?\$\<\>\⊆\∪\∩\`\∫\[\]\^]
```

Comment:
Seems like this is an arbitrary list of symbols. Anyway to include all Unicode chars?